cheri/Makefile

```diff
@@ -128,11 +128,11 @@ cheri.v cheri_types.v: $(CHERI_SAILS)
 	$(SAIL) -coq -o cheri -coq_lib mips_extras -undefined_gen -memo_z3 -dcoq_undef_axioms $^
 
 CHERI_COQ = cheri_types.v mips_extras.v cheri.v
-COQ_LIBS = -R ../../bbv/theories bbv -R ../lib/coq Sail
+COQ_LIBS = -R $(SAIL_DIR)/../bbv/theories bbv -R $(SAIL_DIR)/lib/coq Sail -R ../mips ''
```

Comment:
Where does the bbv directory come from? I think it will not work when `SAIL_DIR` is set to installed share directory of opam package.

Reply:
The Coq stuff isn't set up for opam yet.  I still need to look into how opam packages for coq libraries are set up, and whether there's a bbv one.  At the moment it just assumes that you've cloned all the repos into the same directory.

Reply:
OK. Perhaps we should introduce a `BBV_DIR` make variable which defaults to `$(SAIL_DIR)/../bbv` but can be overridden on command line if necessary?

Reply:
Maybe, but for now I'd rather avoid the impression that builds are fully set up when they're not.